main.py

```diff
@@ -1,5 +1,530 @@
-import moduls.dna_rna_tools
-import moduls.fastq_tools
-import moduls.protein_tools
-
-
+import os
+import sys
+import time
+import datetime
+import requests
+import traceback
+import functools
+from io import StringIO
+from dotenv import load_dotenv
+
+
+def transcribe(sequns: str) -> str:
+    """
+    That function transcribe DNA to RNA
+    :param sequns: DNA sequence
+    :return: RNA sequence
+    """
+    return "".join([transcription_dict[i] for i in sequns])
+
+
+def reverse(sequns: str) -> str:
+    """
+    That function reverses DNA sequence
+    :param sequns:DNA sequense
+    :return: reversed DNA sequence
+    """
+    return sequns[::-1]
+
+
+def complement(sequns: str) -> str:
+    """
+    That function create a complementarity chain fot the DNA sequence
+    :param sequns: DNA sequense
+    :return: complementarity chain to the DNA sequence
+    """
+    return "".join([complement_dict[i] for i in sequns])
+
+
+def reverse_complement(sequns: str) -> str:
+    """
+    That function create reverse complementarity chain for the DNA sequence
+    :param sequns: sequense
+    :return: reversed complimentarity chain to the DNA sequence
+    """
+    rev_sequns = sequns[::-1]
+    return "".join([complement_dict[i] for i in rev_sequns])
+
+
+operations = {
+    "transcribe": transcribe,
+    "reverse": reverse,
+    "complement": complement,
+    "reverse_complement": reverse_complement,
+}
+
+
+transcription_dict = {
+    "A": "U",
+    "a": "u",
+    "T": "A",
+    "t": "a",
+    "G": "C",
+    "g": "c",
+    "C": "g",
+    "c": "g",
+}
+
+
+complement_dict = {
+    "A": "T",
+    "a": "t",
+    "T": "A",
+    "t": "a",
+    "G": "C",
+    "g": "c",
+    "C": "g",
+    "c": "g",
+}
+
+
+def dna_rna_tools(*args):
+    *sequnses, def_name = args
+    for sequns in sequnses:
+        if RNA_checking(sequns) == True:
+            continue
+        else:
+            raise ValueError("Incorrect options input, please try again")

```

Comment:
1) Не надо в проверках булевых значений делать `== True`
2) Не надо делать `continue else raise` - просто добавь `not` в условие

```suggestion
        if not check_rna(sequence):
            raise ValueError("Incorrect options input, please try again")

```